lib/lints/avoid_late_keyword/models/_config_parser.dart

```diff
@@ -0,0 +1,15 @@
+part of 'avoid_late_keyword_parameters.dart';
+
+class _ConfigParser {
+  static const _allowInitializedConfig = 'allow_initialized';
+  static const _ignoredTypesConfig = 'ignored_types';
+
+  static bool parseAllowInitialized(Map<String, Object?> config) =>
+      config[_allowInitializedConfig] as bool? ?? false;
+
+  static Iterable<String> parseIgnoredTypes(Map<String, Object?> config) =>
+      config.containsKey(_ignoredTypesConfig) &&
+              config[_ignoredTypesConfig] is Iterable
+          ? List<String>.from(config[_ignoredTypesConfig] as Iterable)
+          : <String>['AnimationController'];
+}
```

Comment:
Let's make it part of AvoidLateKeywordParameters structure.

And the parseX method can be united into a single constructor/factory `fromJson`